src/views/ont/OntHeronRun.vue

```diff
@@ -88,7 +88,7 @@ export default {
         if (response.errors.length > 0) {
           this.showAlert('Failure: ' + response.errors.join(', '), 'danger')
         } else {
-          this.redirectToRuns()
+          this.showAlert('Run successfully created', 'success')
```

Comment:
Do you also need to update the `run` variable based on the response?

Reply:
Yes. Thought I would wait until the caching issue was sorted because I guess I would be using the same code to update the run variable.